functions/Set-DbaMaxDop.ps1

```diff
@@ -121,11 +121,11 @@ function Set-DbaMaxDop {
 		$collection | Add-Member -Force -NotePropertyName OldInstanceMaxDopValue -NotePropertyValue 0
 		$collection | Add-Member -Force -NotePropertyName OldDatabaseMaxDopValue -NotePropertyValue 0
 		
-		$servers = $collection | Select-Object Instance -Unique
+		$servers = $collection | Select-Object SqlInstance -Unique
 		
 		foreach ($server in $servers) {
-			if ($server.Instance -ne $null) {
-				$servername = $server.Instance
+			if ($server.InstanceName -ne $null) {
```

Comment:
This property will not exist because the `Test-DbaMaxDop` was updated in a previous PR [here](https://github.com/sqlcollaborative/dbatools/pull/2564/files#diff-e2df3864d4a0b7753518746e41ee7969L173).

You need to match up the right property name.

Reply:
So close! What is the right property name, @wsmelton ? I can merge and patch.

Thanks so much, @josh-simar !

Reply:
`SqlInstance`